tapeout/src/test/resources/macros/lib-1024x8-mrw.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "type": "sram",
+    "name": "vendor_sram",
+    "depth": 1024,
+    "width": 8,
+    "ports": [
+      {
+        "clock port name": "clock",
+        "mask granularity": 8,
+        "output port name": "RW0O",
+        "input port name": "RW0I",
+        "address port name": "RW0A",
+        "mask port name": "RW0M",
+        "chip enable port name": "RW0E",
+        "write enable port name": "RW0W",
+        "clock port polarity": "positive edge",
+        "output port polarity": "active high",
+        "input port polarity": "active high",
+        "address port polarity": "active high",
+        "mask port polarity": "active high",
+        "chip enable port polarity": "active high",
+        "write enable port polarity": "active high"
+      }
+    ]
+  },
+  {
+    "type": "metal filler cell",
```

Comment:
Not for the macro compiler, but needed for floorplan, I guess. This also tests how well the macro compiler filters SRAMs only in JSON.